lib/caesar.rb

```diff
@@ -1,6 +1,41 @@
+require 'pry'
 class Caesar
-  attr_reader :key
+  attr_reader  :dictionary, :number
 
   def initialize
+    @dictionary = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", " "]
+    @number = 0
```

Comment:
Why are you defining this here if we just overwrite it later?

Reply:
because the number arguement from  eng_to_cipher(message, number) goes to the method rotate ..after it changes @number from initialize

Reply:
if it changes, you dont need to define it here, and it should not be stored as an attribute